validate.lic

```diff
@@ -395,7 +395,7 @@ class DRYamlValidator
     return unless settings.training_abilities
 
     settings.training_abilities
-            .reject { |skill, _cooldown| ['PercMana', 'Perc', 'Perc Health', 'Astro', 'App', 'App Quick', 'App Careful', 'Tactics', 'Hunt', 'Pray', 'Scream', 'Khri Prowess', 'Stealth', 'Ambush Stun', 'Ambush Choke', 'Favor Orb', 'Charged Maneuver', 'Meraud', 'Recall', 'Barb Research Debilitation', 'Barb Research Augmentation', 'Barb Research Warding', 'Collect', 'Analyze', 'PrayerMat', 'Smite', 'Locks'].include?(skill) }
+            .reject { |skill, _cooldown| ['PercMana', 'Perc', 'Perc Health', 'Astro', 'App', 'App Quick', 'App Careful', 'Tactics', 'Hunt', 'Pray', 'Scream', 'Khri Prowess', 'Stealth', 'Ambush Stun', 'Ambush Choke', 'Favor Orb', 'Charged Maneuver', 'Meraud', 'Recall', 'Barb Research Debilitation', 'Barb Research Augmentation', 'Barb Research Warding', 'Collect', 'Analyze', 'PrayerMat', 'Smite', 'Locks', 'Teach'].include?(skill) }
             .each_key { |skill| error("Ability in training_abilities is not valid: #{skill}") }
```

Comment:
❓ Is there value to a validator that checks that the value of `combat_teaching_skill` is a valid skill? Or warns when it's say, Stealth, Thievery, Sorcery, Arcane, ...?

Reply:
yep good call, I'll make one. Don't think that should block this though.

Reply:
:+1: